### PR TITLE
Actually disable global_uid for internal metadata for Rails 5+

### PR DESCRIPTION
**Pull request by pschambacher**
/cc @bquorning @grosser 
My last PR called the method to check if it was disabled, not the one to actually disable global_uid...

### CODE REVIEW

File: lib/global_uid.rb [hunk 1]
```diff
@@ -16,19 +16,20 @@ class TimeoutException < Exception ; end
 ActiveRecord::Base.send(:include, GlobalUid::ActiveRecordExtension)
 ActiveRecord::Migration.send(:prepend, GlobalUid::MigrationExtension)
 
+# Make sure that GlobalUID is disabled for ActiveRecord's SchemaMigration table
 if defined?(ActiveRecord::SchemaMigration)
   ActiveRecord::SchemaMigration.disable_global_uid
 end
 
+# Make sure that GlobalUID is disabled for ActiveRecord's Internal Metadata table
+if ActiveRecord::VERSION::MAJOR >= 5
```
Comment (bquorning) [inline lib/global_uid.rb]: Why not make this line consistent with line 6?

Comment (pschambacher) [inline lib/global_uid.rb]: Line 6?

Comment (bquorning) [inline lib/global_uid.rb]: I don’t know why my diff tool said line 6. I meant line 20, obviously. Why do we do two equivalent checks in two different ways, right after each other?

Comment (bquorning) [inline lib/global_uid.rb]: To clarify: Why do we say `if defined?(ActiveRecord::SchemaMigration)` above and not ` if defined?(ActiveRecord::InternalMetadata)` here?

Comment (pschambacher) [inline lib/global_uid.rb]: We should probably change the `defined?(ActiveRecord::SchemaMigration)` to use a Rails version. Currently we could have a bug if global_uid is loaded before this class is defined.

**Pull request merged**